### ISSUE TITLE
Missing OIDC group/role bindings on the management cluster (for admin role/first user)

### ISSUE THREAD

**Issue by teemow** [giantswarm/roadmap#734]
While onboarding a new customer we ran into the problem that it was hard to follow the gitops tutorial because the customer didn't have the correct role bindings on the management cluster.

Error:
```
Error from server (Forbidden): error when creating "01-source.yaml": gitrepositories.source.toolkit.fluxcd.io is forbidden: User cannot create resource "gitrepositories" in API group "source.toolkit.fluxcd.io" in the namespace "flux-system"
```

Tutorial:
https://docs.giantswarm.io/advanced/gitops/#setting-up-sources

The solution is to bind the role `write-flux-resources`

![screen_2022-01-21-17-22-46](https://user-images.githubusercontent.com/32488/150565259-00022ed8-bc81-431f-82d6-3e2dee7aca6d.png)

Question is if we add a hint to the tutorial or make sure that the initial user at least get's this permission.

**Comment by kubasobon** [comment]
`flux-demo` repository and docs have been updated. They now use `default` namespace and `automation` ServiceAccount, which allow customers to follow the steps without taking extra action. 
If they still want to set up their own ServiceAccount, an explanation along with link to MAPI RBAC documentation is provided.
